scania-digital-design-system/tegel
# feat(card): align slots

**Describe pull-request**  
Introduces slots for:
 - Header
 - Subheader
 - Thumbnail
 - Body image
 - Body
 - Bottom


BREAKING
Removed the header-img props.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1955

**How to test**  
1. Go to storybook
2. Check in Components -> Card
3. Test the controls and check the styling

**Suggested test steps**
